core/src/avm2/amf.rs

```diff
@@ -281,7 +281,10 @@ pub fn deserialize_value<'gc>(
             }
             let storage = VectorStorage::from_values(
                 vec.iter()
-                    .map(|v| deserialize_value(activation, v))
+                    .map(|v| {
+                        deserialize_value(activation, v)
+                            .map(|value| value.as_object().map(Value::from).unwrap_or(Value::Null))
```

Comment:
I need to fix this to only convert `Undefined` to `Null`- that should stop the failing tests (since there's no `Vector.<void>`)